agent/tags.go

```diff
@@ -143,3 +205,38 @@ func FetchTags(l *logger.Logger, conf FetchTagsConfig) []string {
 
 	return tags
 }
+
+// TODO: Seems maybe el crapola, dunno will ask lox
```

Comment:
Well, is it? 😅

Reply:
You could probably just use `strings.SplitN` to force it to split into two chunks and then check the len is 2.